### DISCUSSION TITLE
DFTD3 errors.

### DISCUSSION THREAD

**Discussion by wuxy92** [dftbplus/dftbplus#1092] [Q&A] [answered]
Dear developers,
I compiled the dftb+ with s-dftd3 library (21.2 version) and performed the calculation including DFTD3 Dispersion.   
However, the calculation was exit with a Segmentation fault. If without DFTD3 correction, the calculation worked fine.  
I'm confused by this error.  Could you give me some advice?

Thanks.

xiaoyan

**Comment by aradi** [comment, marked as answer]
It may be an stack exhaustion problem with OpenMP. Could you try to unlimit the stack size in the shell, where you run DFTB+ from? (e.g. `ulimit -s unlimited`).

**Reply by wuxy92** [reply]
I downloaded it from Github.  And I've used the ulimit -s unlimited command.

**Reply by wuxy92** [reply]
Sorry, I confused version 22.1 with 21.2 and version 22.1 works fine, thanks.

**Reply by aradi** [reply]
Perfect, I am happy to hear, that it works!

**Reply by wuxy92** [reply]
thank,
